tests/Doctrine/Tests/DBAL/DriverManagerTest.php

```diff
@@ -151,27 +151,27 @@ public function databaseUrls()
             ),
             'sqlite relative URL with host' => array(
                 'sqlite://localhost/foo/dbname.sqlite',
-                array('dbname' => 'foo/dbname.sqlite', 'driver' => 'Doctrine\DBAL\Driver\PDOSqlite\Driver'),
+                array('dbname' => 'foo/dbname.sqlite', 'path' => 'foo/dbname.sqlite', 'driver' => 'Doctrine\DBAL\Driver\PDOSqlite\Driver'),
```

Comment:
The original data provider entries test things that don't matter: `dbname` does nothing for SQLite driver. I can create new data provider entries if you'd like, but it seemed that changing the existing ones to test something that mattered made more sense.